### PR TITLE
Release/0.7.1

### PR DESCRIPTION
**Pull request by dilyand**
Snyk integration won't be complete until the repo is set up with an API token.

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,18 @@
+Version 0.7.1 (2020-06-26)
+--------------------------
+Bump geoip2 to 2.13.1 (#136)
+Bump cats-effect to 1.4.0 (#137)
+Bump cats-core to 1.6.1 (#138)
+Bump scala-lru-map to 0.4.0 (#139)
```
Comment (chuwy) [inline CHANGELOG:6]: This will bring cats 2 and cats-effect 2, which makes above bumps useless.

Comment (dilyand) [reply CHANGELOG:6]: I bumped both cats-effect and cats-core to 2.x.x.

**Comment by dilyand** [comment]
@chuwy are you happy with the changes? If yes, leave a green tick please.

Also, is the Snyk token in place?

File: CHANGELOG [hunk 2]
```diff
@@ -1,3 +1,18 @@
+Version 0.7.1 (2020-06-26)
+--------------------------
+Bump geoip2 to 2.13.1 (#136)
+Bump cats-effect to 1.4.0 (#137)
+Bump cats-core to 1.6.1 (#138)
```
Comment (chuwy) [inline CHANGELOG:5]: Items in CHANGELOG are still for 1.x

**Comment by chuwy** [comment]
> Also, is the Snyk token in place?

@dilyand no, it is not. Can you make me a repo moderator to add it?

**Comment by chuwy** [comment]
Added `SNYK_TOKEN`

**Comment by chuwy** [comment]
Also, I think nothing is holding us from adding Scala 2.13. There's a PR (https://github.com/snowplow/scala-maxmind-iplookups/pull/127), but it has no signed CLA and I think it's just one line change, given everything else is updated.

**Pull request merged**